mlrun/runtimes/kubejob.py

```diff
@@ -143,6 +161,10 @@ def deploy(
         """
 
         build = self.spec.build
+
+        # make sure we disable load_on_run mode if the source code is in the image
+        build.load_source_on_run = False
```

Comment:
Not sure I follow, if you build an image, why it means that the source code is in the image ?